### ISSUE TITLE
[Bug]:

### ISSUE THREAD

**Issue by gabrielrussoc** [aspect-build/rules_py#649]
### What happened?

Running the interpreter inside a venv crashes with

```
thread 'main' panicked at py/tools/venv_shim/src/main.rs:199:40:
called `Result::unwrap()` on an `Err` value: RunfilesDirIoError(Os { code: 2, kind: NotFound, message: "No such file or directory" })
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Seems like it was also reported here: https://github.com/aspect-build/rules_py/pull/629#issuecomment-3272165500

### Version

Development (host) and target OS/architectures:
Linux x64

Output of `bazel --version`:
bazel 7.6.1

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:

rules_py 1.6.4-rc1
rues_python 0.31.0

Language(s) and/or frameworks involved:

### How to reproduce

```shell
bazel run //path/to:py_binary.venv



source /home/gabriel.russo/universe/.path+to+py_binary.venv/bin/activate



python3


see


thread 'main' panicked at py/tools/venv_shim/src/main.rs:199:40:
called `Result::unwrap()` on an `Err` value: RunfilesDirIoError(Os { code: 2, kind: NotFound, message: "No such file or directory" })
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
